packages/design-system/src/components/HelpDrawer/useHelpDrawerAnalytics.ts

```diff
@@ -0,0 +1,54 @@
+import { EventCategory, sendLinkEvent, useAnalyticsContent } from '../analytics';
+import { HelpDrawerProps } from './HelpDrawer';
+import { useCallback } from 'react';
+import { helpDrawerSendsAnalytics } from '../flags';
+
+export default function useHelpDrawerAnalytics({
+  analytics,
+  analyticsLabelOverride,
+}: HelpDrawerProps) {
+  const sendHelpDrawerEvent = useCallback(
```

Comment:
Now that I think about it, I don't need `useCallback`, because the function isn't being applied to any components in the subtree.